### PR TITLE
Document our native iOS stack

### PR DESCRIPTION
**Pull request by jferris**

### CODE REVIEW

File: stack/README.md [hunk 1]
```diff
@@ -33,6 +33,28 @@ decisions made for other layers.
...
 
+### iOS (Native)
+
+* Use Swift for writing native app code.
+* Use Xcode and xcodebuild for building iOS apps.
+* Use xcpretty to format xcodebuild output.
+* Use Xcode automatic provisioning when possible.
+* Use UIKit with Storyboards and MVVM for creating UIs.
+* Use Dispatch and OperationQueue for concurrency.
+* Use VoiceOver and VoiceControl for accessibility.
```
Comment (sharplet) [inline stack/README.md:44]:
```suggestion
* Support VoiceOver and VoiceControl for accessibility.
```

File: stack/README.md [hunk 2]
```diff
@@ -33,6 +33,28 @@ decisions made for other layers.
...
+* Use Swift Package Manager for dependencies when possible.
+* Manually test on both iPhone and iPad to ensure the app is functional.
+* Work closely with designers on UI components
```
Comment (sharplet) [inline stack/README.md:47]:
```suggestion
* Work closely with designers on UI components.
```

File: stack/README.md [hunk 3]
```diff
@@ -33,6 +33,28 @@ decisions made for other layers.
 * Use Gradle with Android Studio for building.
 * Use MVVM to model views.
 
+### iOS (Native)
+
+* Use Swift for writing native app code.
+* Use Xcode and xcodebuild for building iOS apps.
+* Use xcpretty to format xcodebuild output.
+* Use Xcode automatic provisioning when possible.
+* Use UIKit with Storyboards and MVVM for creating UIs.
```
Comment (das3in) [inline stack/README.md:42]: I've been trying to brush up on my iOS development and ran through some SwiftUI stuff and thought the experience was pleasant - I'm not a dedicated mobile developer but I'd endorse using SwiftUI in the future

Comment (jferris) [reply stack/README.md:42]: My understanding is SwiftUI is new enough that it's still an "experiment" for us.

**Pull request merged**